docs/References.bib

```diff
@@ -355,6 +355,27 @@ @article{Lovelace2008tw
   SLACcitation   = "%%CITATION = ARXIV:0805.4192;%%"
 }
 
+@article{Lovelace2016uwp,
+      author         = "Lovelace, Geoffrey and  Lousto, Carlos O. and
+ Healy, James  and  Scheel, Mark A. and  Garcia, Alyssa and
+ O'Shaughnessy, Richard and Boyle, Michael  and Campanelli, Manuela and
+ Hemberger, Daniel A. and Kidder, Lawrence E. and Pfeiffer, Harald P. and
+ Szilágyi, B\'{e}la and Teukolsky, Saul A. and Zlochower, Yosef",
```

Comment:
Please look at the other bibliography entries and format this the same. In particular, fix the indentation please.

Reply:
Hello, I think I fixed the issue.  I don't expect you to take a look at it right away because I know you are super busy; but if there is anything wrong with it when you get a chance to look at it, please let me know. Thank you for your feed back! 
:)